types/mui-datatables/index.d.ts

```diff
@@ -70,7 +70,7 @@ export interface MUIDataTableMeta {
     columnIndex: number;
     rowData: any[];
     rowIndex: number;
-    tableData: MUIDataTableData[];
+    tableData: any[];
```

Comment:
```suggestion
    tableData: T[];
```

And change the interface to 
```ts
export interface MUIDataTableMeta<T = any>
```

That'll allow better type resolutions for the `tableData`

Reply:
thank you for your suggestion! i just pushed the changes :)